zipline-loader/src/test/kotlin/app/cash/zipline/loader/ziplineManifestTest.kt

```diff
@@ -0,0 +1,146 @@
+package app.cash.zipline.loader
+
+import com.google.common.truth.Truth.assertThat
+import kotlin.test.assertFailsWith
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import okio.ByteString.Companion.encodeUtf8
+import org.junit.Test
+
+// for some reason naming this file ZiplineManifestTest.kt breaks this file in Intellij
```

Comment:
does it break outside of intellij?

Reply:
There’s a git + filesystems thing where case insensitive file systems will fight you if you rename a file by changing its case only.

Reply:
Probably works best if you rename to `Foo.kt` and then rename again to the name you want

Reply:
1. Use a case-sensitive file system for your code!
2. This is not actually a problem in IntelliJ anymore: https://blog.jetbrains.com/idea/2016/06/how-to-support-case-only-rename-in-git/

Reply:
I've tried renaming to other things and back, deleting the file, recreating, emptying caches to no avail.